uber/templates/groups/form.html

```diff
@@ -183,13 +183,57 @@
             <textarea rows="3" cols="30" name="special_needs">{{ group.special_needs }}</textarea>
         </div>
     </div>
```

Comment:
It gives a warning to the admin - I haven't tested this explicitly, but you
should be able to go into the group leader's details and edit their address
there.

On Tue, Jul 28, 2015, 22:57 Eli Courtwright <notifications@github.com>
wrote:

> In uber/templates/groups/form.html
> <https://github.com/magfest/ubersystem/pull/1289#discussion_r35724144>:
>
> > @@ -183,13 +183,57 @@
> >              <textarea rows="3" cols="30" name="special_needs">{{ group.special_needs }}</textarea>
> >          </div>
> >      </div>
>
> This suppresses these form fields in cases where the leader doesn't
> already have an address filled out. Won't that make it impossible for an
> admin to enter this information if it's not already entered?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/magfest/ubersystem/pull/1289/files#r35724144>.
>